src/Network/GRPC/HighLevel/Generated.hs

```diff
@@ -28,10 +28,17 @@ module Network.GRPC.HighLevel.Generated (
 
   -- * Server Auth
 , ServerSSLConfig(..)
+
+  -- * Client
```

Comment:
Hmmm, that's one _downside_ to having all the low level and high level library surfaces exposed from the same package; I could imagine splitting off the low level surface into `gRPC-haskell-lowlevel` and tracking the gRPC C core's version number with that package.

Then the `gRPC-haskell` package could maintain its own versioning semantics to communicate updates or changes to its APIs which may have convenience features added out-of-band from gRPC C core changes.

But, I think that's more work than is worth even considering right now, I think documenting it like you have is great.